lib/sidebar-manager.js

```diff
@@ -414,16 +414,16 @@ exports.sidebarManager = {
     },
     loadNext: function() {
         var lastCTATime = storageManager.get('lastSidebarCTACompleteTime');
-
         // get the time elapsed since the last cta completion
-        var timeElapsed = utils.getTimeElapsed(lastCTATime);
-
-        // get the amount we need to add to make up for the difference between CTA completion and when the timer should be done
-        var timeElapsedDiff = (intervals.defaultSidebarInterval - utils.getTimeElapsed(lastCTATime) > 0 ? intervals.defaultSidebarInterval - utils.getTimeElapsed(lastCTATime) : 0);
+        var timeElapsed = parseInt(utils.getTimeElapsed(lastCTATime));
```

Comment:
So I can use the return value here:
`storageManager.set('lastSidebarCTACompleteTime', utils.getNextCTACompleteTime(...));`